charts/hmc-cft-hearing-service/values.preview.template.yaml

```diff
@@ -22,7 +22,8 @@ java:
     HMI_SOURCE_SYSTEM: CFT
     HMI_DESTINATION_SYSTEM: SNL
     GRANT_TYPE: client_credentials
-    FH_BASE_URL: https://login.microsoftonline.com
+    FH_BASE_URL: http://ccd-test-stubs-service-aat.service.core-compute-aat.internal
```

Comment:
Duplicate keys found.Does  FH_BASE_URL should be added here? is this the correct value? I guess FH_BASE_URL should be https://login.microsoftonline.com.

FH_BASE_URL: http://ccd-test-stubs-service-aat.service.core-compute-aat.internal (line 89)
FH_BASE_URL: https://login.microsoftonline.com (line 25)

HMI_BASE_URL should be  http://ccd-test-stubs-service-aat.service.core-compute-aat.internal/hmi (line 85) which is correct and configured in line 85

Reply:
Additional FH_BASE_URL and HMI_BASE_URL environment variables have been removed.  This was done in #636 which reverted changes to file that added them.